db/migrations/dedupe_github_repo_owner.py

```diff
@@ -0,0 +1,61 @@
+"""Remove case-sensitive duplicates in GitHub repository tables"""
+
+import rethinkdb as r
+
+import db.plugins
+import db.util
+
+r_conn = db.util.r_conn
+
+
+if __name__ == '__main__':
+    table = 'plugin_github_repos'
+
+    print 'Removing duplicate rows in %s' % table
+
+    updated = 0
+    deleted = 0
+
+    query = r.table(table)
+
+    # Group by the normalized GitHub path
+    query = query.group([
+        r.row['owner'].downcase(),
+        r.row['repo_name'].downcase()])
+
+    # Get the most recently scraped row in each group first
+    query = query.order_by(r.desc('last_scaped_at'))
+
+    grouped_repos = query.run(r_conn())
+
+    for owner_repo, repos in grouped_repos.iteritems():
+
+        print '\nRepo with GitHub path %s occurs %s times' % (
+                owner_repo,
+                len(repos))
+
+        # Use the most recently scraped row as the canonical row
+        canonical = repos.pop()
+
+        assert canonical
```

Comment:
Good point. I had honestly been using this as part of my debug process while building the script. In that case, I was working with a database dump which I could easily restore. Obviously I'll make a backup of our production DB before running this script (we make one every day anyway), so I could always restore from that if needed.

I'll leave it, since I'm 99.99% sure it could never be falsy anyway (given the way the rethinkdb api works).

Good things to consider though. I'm still formulating for myself what the proper way/time to use `assert` actually is.